frontend/src/scenes/settings/project/PersonsOnEvents.tsx

```diff
@@ -62,8 +62,8 @@ export function PersonsOnEvents(): JSX.Element {
 
     return (
         <>
-            <p>Choose how to query your event data with person filters.</p>
-            <LemonRadio value={poeMode} onChange={setPoeMode} options={poeOptions} />
+            <p>Choose the behavior of person property filters.</p>
```

Comment:
```suggestion
            <p>Choose how properties relating to a person change in comparison to the events setting them.</p>
```

Reply:
I feel like this is a very complex sentence 😅 but it is a complex setting. Actually, I just recalled Tiina made a docs page about this. Added a link to it for those interested in the specifics of "behavior" here:
<img width="559" alt="Screenshot 2024-06-14 at 11 01 44" src="https://github.com/PostHog/posthog/assets/4550621/fb664b2f-9065-46c6-a6e8-e64d9347cd65">

Reply:
I guess my main point here is that this doesn't only affect filters, but everything related to person properties. So maybe just "Choose the behavior of person properties."?

Docs link is great 👍

Reply:
Hmm, is there something other than filters affected? So, what I'm thinking is that it's actually a useful distinction to clarify whether this setting is query-time or ingestion-time, and it's the former – so saying it's about the "filters" might be useful in that sense, because it's clear the data stays the same, it's just a different way of querying it

Reply:
> Hmm, is there something other than filters affected?

Breakdowns, aggregations, etc.

>  So, what I'm thinking is that it's actually a useful distinction to clarify whether this setting is query-time or ingestion-time,

Yup, super useful! We could also adapt the header to say "Person properties querying" instead of the current "Person properties mode".

> so saying it's about the "filters" might be useful in that sense, because it's clear the data stays the same, it's just a different way of querying it

In any case happy for this to go in.